src/components/Repo/RepoDetails.jsx

```diff
@@ -242,7 +242,29 @@ function RepoDetails() {
       : `Timestamp N/A`;
     return lastDate;
   };
-
+  const handleDeleteRepo = () => {
+    const confirmed = window.confirm('Are you sure you want to delete this repo?');
+    if (confirmed) {
+      const apiUrl = `http://localhost:3000/v2/${name}/manifests/`;
```

Comment:
We declare our endpoints inside the api.js file. and send requests using axios not fetch. Check the api.js file and usage of api thought the project for examples.
Also, the backend host is declared in the hosts.js file, don't hardcode it inside code